### PR TITLE
Fixes #404 Provide Grafana dashboard for metrics

### LINKED ISSUE #404: Provide grafana dashboard and report more metrics
https://issues.apache.org/jira/browse/ACCUMULO-4850

PR #403

### PR DESCRIPTION
**Pull request by mikewalch**
* Also, enabled more reporting of ingest/scan metrics

### CODE REVIEW

**Comment by mikewalch** [comment]
The metrics can be set up quickly using `uno setup accumlo --with-metrics` in this PR https://github.com/astralway/uno/pull/173

**Comment by mikewalch** [comment]
PR for instructions for setting up metrics in Grafana/InfluxDB: https://github.com/apache/accumulo-website/pull/62

**Comment by mikewalch** [comment]
From review of apache/accumulo-website#62, I am going to move the grafana-dashboard.json out of this PR.  Not 100% sure where yet.

File: assemble/pom.xml [hunk 1]
```diff
@@ -274,6 +274,15 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.apache.rat</groupId>
+        <artifactId>apache-rat-plugin</artifactId>
+        <configuration>
+          <excludes>
+            <exclude>conf/templates/grafana-dashboard.json</exclude>
```
Comment (ctubbsii) [inline assemble/pom.xml]: This isn't needed anymore.

Comment (mikewalch) [reply assemble/pom.xml]: It has been removed.

**Pull request merged**